home/tests/tests.py

```diff
@@ -157,3 +164,7 @@ def test_half_custom_image_within_preset(self):
         self.assertEqual(int(rendered_image.get('width')), image_rendition.width)
         self.assertEqual(int(rendered_image.get('height')), image_rendition.height)
         self.assertEqual(rendered_image.get('src'), image_rendition.url)
+
+
+def parse_html(html: str) -> BeautifulSoup:
```

Comment:
I know it's a one-liner function but I prefer to move this function to a more generic/shared place so other apps can utilize it without defining it as their own, which will keep it DRY and manageable.

Reply:
I agree that this should be in a single, shared place, but I am not sure where that place is. The function is required by the 'questionnaires' and 'home' apps; do we keep it in 'home' and create a dependency on it in 'questionnaires'? Or, would you suggest something else?

Reply:
`iogt/` project directory is the common place to hold these shared utils.

Reply:
Technically, that would work, but it would violate the principle of the project depending on the apps and not the other way around. The home and questionnaires apps would not be reusable outside of the iogt project. In theory, someone might want a version of IoGT without the questionnaire stuff.

I think questionnaires really can't exist without the home app, but home should be able to exist by itself, so that makes it a better location for shared code. A problem would arise if `parse_html` was needed by an app that doesn't currently depend on home, but perhaps we cross that bridge when we get to it.

Reply:
Resolved by removing `parse_html` in `questionnaires` and keeping it in `home`.